app/src/main/java/ar/com/wolox/lucasdelatorre/training/activities/Login.java

```diff
@@ -96,21 +105,33 @@ private boolean validateInputs() {
         return true;
     }
 
-    private void saveCredentials(String username, String password) {
-        //TODO: Save credentials
+    private void saveCredentials() {
+        SharedPreferences sharedPref = getPreferences(Context.MODE_PRIVATE);
+        SharedPreferences.Editor editor = sharedPref.edit();
+        editor.putString(getString(R.string.login_username_key), mUser.getUsername());
+        editor.putString(getString(R.string.login_sessiontoken_key), mUser.getSessionToken());
+        editor.commit();
     }
 
-    //TODO: implement it
-    private boolean isLogged()
-    {
-        return false;
+    private void checkCredentials() {
+        if (!validateInputs()) return;
+
+        LoginService loginTry = RestApiAdapter.getAdapter().create(LoginService.class);
+        loginTry.login(mUsername, mPassword, mLoginCallback);
+    }
+
+    private void isLogged() {
+        SharedPreferences sharedPref = getPreferences(Context.MODE_PRIVATE);
+        String token = sharedPref.getString(getString(R.string.login_sessiontoken_key), "");
```

Comment:
![giphy](https://cloud.githubusercontent.com/assets/494001/7209069/749f5758-e51b-11e4-99b1-572196b01fe1.gif) line to long :cactus: